pkg/actors/internal/placement/placement.go

```diff
@@ -129,7 +129,7 @@ func New(opts Options) (Interface, error) {
 		hostname:      opts.Hostname,
 		operationLock: fifo.New(),
 		apiLevel:      opts.APILevel,
-		htarget:       opts.Healthz.AddTarget(),
+		htarget:       opts.Healthz.AddTarget("placement-service"),
```

Comment:
I see `placement-service` twice - is that intentional? Or should this be called:
```suggestion
		htarget:       opts.Healthz.AddTarget("internal-placement-service"),
```